spec/acceptance/zz281_spec.rb

```diff
@@ -164,9 +164,9 @@ class { 'erlang': epel_enable => true}
     describe file('/var/lib/rabbitmq/rabbitmqadmin') do
       it { should be_file }
     end
-
-    describe command('rabbitmqadmin --help') do
-      it { should return_exit_status 0 }
+    
+    describe command('/usr/local/bin/rabbitmqadmin --help') do
```

Comment:
Was the path not working properly? This part seemed to work for me. If it's not working it might be indicative of a bug in the module and not the test.